[Bug]: Clock skew adjuster does not adjust EndTimestamp in ptrace model

### What happened?

The clock skew adjuster in `adjustTimestamps` adjusts `StartTimestamp` and event timestamps but not `EndTimestamp`.


As in v1, this worked correctly because `model.Span` used `StartTime + Duration` adjusting `StartTime` preserved duration.
```
// v1 model (jaeger-idl/model/v1/model.pb.go)
StartTime time.Time     // field 6
Duration  time.Duration // field 7
```

In v2, `ptrace.Span` uses `pcommon.Timestamp` for both `StartTimestamp` and `EndTimestamp` with no `Duration field`. When only `StartTimestamp` is adjusted, `EndTimestamp - StartTimestamp` changes, potentially producing negative durations.

### Steps to reproduce

Adding EndTimestamp assertion to existing test in clockskew_test.go:

expectedEnd := toTime(proto.adjusted + proto.duration).UTC()
assert.Equal(t, expectedEnd, span.EndTimestamp().AsTime(),
    "adjusted end time of span[ID = %d]: duration should be preserved", id)

Results in 3 failing tests:
- adjust child starting before parent
- adjust child starting before parent even if it is longer
- adjust child ending after parent but being shorter

### Expected behavior

EndTimestamp should be adjusted by the same delta as StartTimestamp 
to preserve span duration. Both timestamps originate from the same 
clock and should receive the same correction.

### Relevant log output

```shell
--- FAIL: TestClockSkewAdjuster/adjust_child_starting_before_parent (0.00s)
--- FAIL: TestClockSkewAdjuster/adjust_child_starting_before_parent_even_if_it_is_longer (0.00s)
--- FAIL: TestClockSkewAdjuster/adjust_child_ending_after_parent_but_being_shorter (0.00s)
FAIL
```

### Screenshot

_No response_

### Additional context

Introduced in #6392 when porting clock skew adjuster from v1 to v2 ptrace.
Related: #8165 (abnormal trace duration display)

I can open a PR with the fix if you agree with the analysis.

--- a/cmd/jaeger/internal/extension/jaegerquery/internal/adjuster/clockskew.go
+++ b/cmd/jaeger/internal/extension/jaegerquery/internal/adjuster/clockskew.go
@@ -192,6 +192,7 @@ func (a *clockSkewAdjuster) adjustTimestamps(n *node, skew clockSkew) {
 		return
 	}
 	n.span.SetStartTimestamp(pcommon.NewTimestampFromTime(n.span.StartTimestamp().AsTime().Add(skew.delta)))
+	n.span.SetEndTimestamp(pcommon.NewTimestampFromTime(n.span.EndTimestamp().AsTime().Add(skew.delta)))
 	jptrace.AddWarnings(n.span, fmt.Sprintf("This span's timestamps were adjusted by %v", skew.delta))
 	for i := 0; i < n.span.Events().Len(); i++ {
 		event := n.span.Events().At(i)

--- a/cmd/jaeger/internal/extension/jaegerquery/internal/adjuster/clockskew_test.go
+++ b/cmd/jaeger/internal/extension/jaegerquery/internal/adjuster/clockskew_test.go
@@ -208,6 +208,11 @@ func TestClockSkewAdjuster(t *testing.T) {
 				assert.Equal(
 					t, toTime(proto.adjusted).UTC(), span.StartTimestamp().AsTime(),
 					"adjusted start time of span[ID = %d]", id)
+
+				assert.Equal(
+					t, toTime(proto.adjusted+proto.duration).UTC(), span.EndTimestamp().AsTime(),
+					"adjusted end time of span[ID = %d]", id)
+
 				for i, logTs := range proto.adjustedEvents {
 					assert.Equal(
 						t, toTime(logTs).UTC(), span.Events().At(i).Timestamp().AsTime(),

